### PR TITLE
Private Cloud release notes June 29

### PR DESCRIPTION
**Pull request by zlogic**
Updated documentation and release notes for the upcoming Private Cloud release.

### CODE REVIEW

**Review by MarkvanMents — APPROVED** [review]
Just one question about use of `MendixApp` but can be published and that can be picked up later if necessary.

File: content/developerportal/deploy/private-cloud-cluster.md [hunk 1]
```diff
@@ -911,15 +923,45 @@ spec:
...
+
 You can change the following options:
 
-* **type**: – select the Endpoint type, possible options are `ingress` and `openshiftRoute`; this parameter is also configured through the **Reconfiguration Script**
+* **type**: – select the Endpoint type, possible options are `ingress`, `openshiftRoute` and `service`; this parameter is also configured through the **Configuration Tool**
 * **ingress**: - specify the Ingress configuration, required when **type** is set to `ingress`
 * **openshiftRoute**: - specify the OpenShift Route configuration, required when **type** is set to `openshiftRoute`
-* **annotations**: - optional, can be used to specify the Ingress or OpenShift Route annotations
+* **annotations**: - optional, can be used to specify the Ingress or OpenShift Route annotations, can be a template: `{{.Name}}` will be replaced with the name of the `MendixApp` CR, and {{.Domain}} will be replaced with the application's domain name
+* **serviceAnnotations**: - optional, can be used to specify the Service annotations, can be a template: `{{.Name}}` will be replaced with the name of the `MendixApp` CR, and {{.Domain}} will be replaced with the application's domain name
```
[the remark this answers is not in the document]
Comment (zlogic) [reply content/developerportal/deploy/private-cloud-cluster.md:956]: Yes, I think it's clearer this way.

**Comment by MarkvanMents** [comment]
@zlogic 
Hi Dmitrii,
Can you let me know when this can be published?
Thanks.

**Comment by zlogic** [comment]
Thank you @MarkvanMents!

We've released the new Mendix for Private Cloud version, can you please publish the release notes now?

**Pull request merged**